automation/taskcluster/decisionlib.py

```diff
@@ -282,6 +287,14 @@ def build_worker_payload(self):
             "artifact_id": self.artifact_id,
         }
 
+        # XXX: Beetmover jobs that transfer the `forUnitTests` maven.zip need
```

Comment:
Hmm.  This really tells me that we can't exploit the richness of Maven.  I wish this was more clear; as a consumer of `m.m.o`, it's weird to me that you're not "Just" publishing my ZIPed directory.  That is, I could have POMs, BOMs, classifiers, extensions... and `m.m.o` would completely fail.  I don't really know what the action is here: I've never seen meaningful docs for `m.m.o`, so even documenting wouldn't actually help a consumer like me.